tools/mdm/apple/getaccount/main.go

```diff
@@ -0,0 +1,89 @@
+package main
```

Comment:
I'm sure I must be missing something, but I couldn't find how/where this is being used in the PR. Would you mind pointing me in the right direction?

Reply:
That's just a script/tool to print out the account information associated with a token, I needed it to verify this: https://github.com/fleetdm/fleet/issues/7515#issuecomment-1330889768 and I thought it could be useful to keep around as a tool. I'll add a comment to explain its use though.